pkgs/applications/office/foliate/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, fetchFromGitHub, meson, ninja, gettext, pkgconfig, python3
+, wrapGAppsHook, gobject-introspection
+, gjs, gtk3, gsettings-desktop-schemas, webkitgtk, glib
+, desktop-file-utils
+/*, hyphen */
+, dict }:
+
+stdenv.mkDerivation rec {
+  pname = "foliate";
+  version = "1.4.0";
+
+  # Fetch this from gnome mirror if/when available there instead!
+  src = fetchFromGitHub {
+    owner = "johnfactotum";
+    repo = pname;
+    rev = version;
+    sha256 = "0qq27bkr8bblsasxw3yll76x0qg7v1x8a70c6971gnc181dakskp";
+  };
+
+  nativeBuildInputs = [
```

Comment:
I'm seeing a `hicolor/icon-theme.cache` in the output.
Can you add `hicolor-icon-theme` so it gets removed by the setup-hook?